apps/ejabberd/src/mongoose_client_api_rooms.erl

```diff
@@ -122,6 +121,16 @@ handle_request(<<"POST">>, JSONData, Req,
             {true, RespReq, State}
     end.
 
+handle_request_by_meth(<<"POST">>, JSONData, User, Server) ->
```

Comment:
I think that semantically both "create room" and "create identifiable room" should use `PUT`.

Actually why aren't we reusing the same method name with `id` being optional? IMO "create identifiable room" is a bit unintuitive.